### PR TITLE
Update builder to v0.0.6 and include controller-gen install

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Add controller-gen binary to builder image so we can use it.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/build/docker/builder/Dockerfile [hunk 1]
```diff
@@ -77,11 +77,15 @@ RUN \
     ( cd $GOPATH/src/k8s.io/kube-openapi/cmd/openapi-gen && \
     git checkout 30be4d16710ac61bce31eb28a01054596fe6a9f1 && \
     go install ./... ) && \
+    (go get -u -d sigs.k8s.io/controller-tools/cmd/controller-gen && \
+    cd $GOPATH/src/sigs.k8s.io/controller-tools && \
+    go install ./...) && \
```
Comment (mhenriks) [inline hack/build/docker/builder/Dockerfile:82]: checkout a tag/commit here

Comment (awels) [reply hack/build/docker/builder/Dockerfile:74]: Done

**Pull request merged**